### ISSUE TITLE
Create a simple subscription confirmation page

### ISSUE THREAD

**Issue by jonotron** [TwoStoryRobot/twostoryrobot.com#118]
When we subscribe someone to the mailing list, it would be nice to send them to a custom subscription confirmation page.

Currently, we will send them here: https://app.convertkit.com/confirm-subscription

Can we create something similar on our own site

**Comment by okanji** [comment]
I think it would be nice if:
- The conformation page is on our website
- We have our logo on this page

**Comment by jonotron** [comment]
Agreed! I think we would probably want this page to be not-indexable by search bots. Not sure if that matters or not or what the general best practice is.
